packages/react-playground/package.json

```diff
@@ -14,10 +14,12 @@
     "stylelint": "stylelint \"src/**/*.scss\"",
     "test": "razzle test --env=jsdom",
     "test:coverage": "razzle test --env=jsdom --silent --coverage",
+    "test:report": "codecov -F react-playground",
     "report": "WITH_REPORT=true razzle build"
   },
   "dependencies": {
     "@logto/react": "^0.0.1",
+    "codecov": "^3.8.3",
```

Comment:
same

Reply:
interesting, run `lerna add --save-dev` seems not recognized as dev dependency.